build.gradle

```diff
@@ -14,117 +14,118 @@
 
 
 plugins {
-	id 'java-library'
-	id 'com.diffplug.spotless' version '5.8.2'
-	id 'maven-publish'
-	id 'signing'
+    id 'java-library'
+    id 'com.diffplug.spotless' version '5.8.2'
+    id 'maven-publish'
+    id 'signing'
 }
 
 group "software.amazon.cloudwatchlogs"
 
 allprojects {
-	compileJava {
-		sourceCompatibility = '1.8'
-		targetCompatibility = '1.8'
-	}
+    compileJava {
+        sourceCompatibility = '1.8'
+        targetCompatibility = '1.8'
+    }
 
-	version = '1.0.4'
+    version = '1.0.4'
 }
 
 java {
-	withJavadocJar()
-	withSourcesJar()
+    withJavadocJar()
+    withSourcesJar()
 }
 
 repositories {
-	jcenter()
-	mavenCentral()
+    jcenter()
+    mavenCentral()
 }
 
 sourceSets {
-	integrationTest {
-		java {
-			compileClasspath += main.output + test.output
-			runtimeClasspath += main.output + test.output
-			srcDir file('src/integration-test/java')
-		}
-		resources.srcDir file('src/integration-test/resources')
-	}
+    integrationTest {
+        java {
+            compileClasspath += main.output + test.output
+            runtimeClasspath += main.output + test.output
+            srcDir file('src/integration-test/java')
+        }
+        resources.srcDir file('src/integration-test/resources')
+    }
 }
 
 configurations {
-	integrationTestImplementation.extendsFrom testImplementation
+    integrationTestImplementation.extendsFrom testImplementation
 }
 
 dependencies {
-	annotationProcessor 'org.projectlombok:lombok:1.18.12'
-
-	compileOnly 'org.projectlombok:lombok:1.18.12'
-	implementation 'com.fasterxml.jackson.core:jackson-core:2.11.1'
-	implementation 'com.fasterxml.jackson.core:jackson-annotations:2.11.1'
-	implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.11.1'
-	implementation 'org.slf4j:slf4j-api:1.7.30'
-
-	// Use JUnit test framework
-	testImplementation 'software.amazon.awssdk:cloudwatch:2.13.54'
-	testImplementation 'junit:junit:4.13'
-	testImplementation 'org.apache.commons:commons-lang3:3.10'
-	testImplementation "org.mockito:mockito-core:2.+"
-	testImplementation "org.powermock:powermock-module-junit4:2.0.2"
-	testImplementation "org.powermock:powermock-api-mockito2:2.0.2"
-	testImplementation "com.github.javafaker:javafaker:1.0.2"
-	testImplementation "com.github.tomakehurst:wiremock-jre8:2.27.0"
-	testImplementation 'software.amazon.awssdk:cloudwatch:2.13.54'
-	testCompileOnly 'org.projectlombok:lombok:1.18.12'
-	testAnnotationProcessor 'org.projectlombok:lombok:1.18.12'
+    annotationProcessor 'org.projectlombok:lombok:1.18.12'
+
+    compileOnly 'org.projectlombok:lombok:1.18.12'
+    implementation 'com.fasterxml.jackson.core:jackson-core:2.11.1'
+    implementation 'com.fasterxml.jackson.core:jackson-annotations:2.11.1'
+    implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.11.1'
+    implementation 'org.slf4j:slf4j-api:1.7.30'
+    implementation 'org.javatuples:javatuples:1.2'
```

Comment:
This is the only change. I'll revert this and fix whatever whitespace issue is happening.